lms/djangoapps/certificates/api.py

```diff
@@ -417,6 +417,12 @@ def _course_from_key(course_key):
 
 
 def _certificate_html_url(user_id, course_id, uuid):
+    """
+    :param user_id:
+    :param course_id:
+    :param uuid:
+    :return:
+    """
```

Comment:
Why did you add docstrings that contain no information?  Why did the reviewer approve them?

Reply:
+1 on this question.

Reply:
@nedbat Failed to see that and thanks for raising it. This PR would be reverted eventually as the issue is resolved. @adeelehsan When you are reverting this?

Reply:
@adeelkhan changes has been reverted https://github.com/edx/edx-platform/pull/19996